Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+def examples = [
+    'hello-world', 
+    'ping-ipv6',
+]
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: '26', description: 'Set 6LoWPAN Channel', name: 'CHANNEL'),
+        stringParam(defaultValue: '0xabcd', description: 'Set 6LoWPAN PAN ID', name: 'PAN_ID'),
+        stringParam(defaultValue: '1', description: 'Set Serial port to use for console', name: 'PADS'),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest"
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: true,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                    [$class: 'SubmoduleOption', recursiveSubmodules: true]
+                ],
+            ])
+        }
+        stage('Build') {
+            for (item in examples) {
+                sh "cd examples/${item} && make TARGET=mikro-e USE_CA8210=1 CHANNEL=${params.CHANNEL} \
+                PAN_ID=${params.PAN_ID} USE_SERIAL_PADS=${params.PADS}"
+                sh "cd examples/${item} && xc32-bin2hex *.mikro-e && mv *.hex ${WORKSPACE}/"
```

Comment:
But the work space is getting cleared every time.

Actually the name of binary file is not same as the directory always. So I have to use * instead of the name.